metadata-db-app/src/cmr/metadata_db/services/subscription_cache.clj

```diff
@@ -0,0 +1,49 @@
+(ns cmr.metadata-db.services.subscription-cache
+  "Defines common functions and defs for the subscription cache.
+	Structure of the hash-cache is as follows:
+	<collection-concept-id> --> <ingest subscription map>
+
+	Example:
+	'C12344554-PROV1' --> { 'enabled' true|false
+                          'mode'    New|Update|Delete|All} "
```

Comment:
Is this choosing a single mode, or is this more like flags? Can a mode be set for New AND Update, or would that need to be set to All to get both events?

Reply:
Right now it is single mode. You can only choose 1.  I didn't think about choosing multiple.  In the API I also have it set to 1. You could create 2 subscriptions, but it would be better to allow multiple.